.ci/pipeline_definitions

```diff
@@ -0,0 +1,52 @@
+gardener-extension-registry-cache:
+  template: 'default'
+  base_definition:
+    repo: ~
+    traits:
+      version:
+        preprocess: 'inject-commit-hash'
+      publish:
+        oci-builder: 'kaniko'
+        dockerimages:
+          gardener-extension-registry-cache:
+            registry: 'gcr-readwrite'
+            image: 'eu.gcr.io/gardener-project/gardener/extensions/registry-cache'
+            dockerfile: 'Dockerfile'
+            target: registry-cache
+          gardener-extension-registry-cache-admission:
+            registry: 'gcr-readwrite'
+            image: 'eu.gcr.io/gardener-project/gardener/extensions/registry-cache-admission'
+            dockerfile: 'Dockerfile'
+            target: registry-cache-admission
+  jobs:
+    head-update:
+      traits:
+        component_descriptor: ~
+        draft_release: ~
+        options:
+          public_build_logs: true
+    release:
+      steps:
+        verify:
```

Comment:
Nice catch! I now added the missing file.

PS. It can be the case that I missed other things as well. Unfortunately, there is no easy way to check this configuration locally to verify it. If I missed other things and we discover them post-merge, I will take care to fix them.